components/scream/src/share/util/scream_common_physics_functions.hpp

```diff
@@ -0,0 +1,163 @@
+#ifndef COMMON_PHYSICS_FUNCTIONS_HPP
+#define COMMON_PHYSICS_FUNCTIONS_HPP
+
+#include "physics/share/physics_constants.hpp"
+#include "share/util/scream_column_ops.hpp"
+
+namespace scream {
+
+template <typename DeviceT>
+struct PhysicsFunctions
+{
+
+  using Device = DeviceT;
+  using KT = KokkosTypes<Device>;
+  using MemberType = typename KT::MemberType;
+
+  template <typename S>
+  using view_1d = typename KT::template view_1d<S>;
+
+  //--------------------------------------------------------------------//
+  //  Functions
+  //--------------------------------------------------------------------//
+
+  // Computes exner function
+  // The result is exners formula, and is dimensionless
```

Comment:
These comments are great. I like to use the square bracket convention for units, which seems to be used elsewhere in physics and earth science, and looks like this:

```
// The input is mid-level pressure [Pa]
```

Reply:
Why `get_exner`, though? Why not just `exner`, given that this is already within the `PhysicsFunctions` struct/"namespace"?